CONTRIBUTING.md

```diff
@@ -91,10 +91,10 @@ Kindly give your branch a more descriptive name like `feat-optimize-images` inst
 
 You could follow this convention. Some ideas to get you started:
 
-- Feature Updates: `feat-<2-3-Words-Description>-<ISSUE_NO>`
-- Bug Fixes: `fix-<2-3-Words-Description>-<ISSUE_NO>`
-- Documentation: `docs-<2-3-Words-Description>-<ISSUE_NO>`
-- And so on...
+*   Feature Updates: `feat-<2-3-Words-Description>-<ISSUE_NO>`
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
* Feature Updates: `feat-<2-3-Words-Description>-<ISSUE_NO>`
```